scripts/script.js

```diff
@@ -0,0 +1,162 @@
+let tasks = [];
+
+// Get Time and date in the required format  27/07/2022 @ 9:38 pm
+let getTime = () => {
```

Comment:
When creating arrow functions, using `let` is not semantically correct. It's better to create arrow functions with `const` because you do not change the value of the function.